### PR TITLE
WDPD-143: Improve shown payment status

### PR DESCRIPTION
**Pull request by ps-amaureder**
### This PR

* The payment status is currently shown as failed till a success notification is received
* After this PR the payment status is pending till a success notification is received

### How to Test

* Pay with SEPA DD, the shown payment status should be pending
* Pay with Paypal, the shown payment status should be success
* Refund a payment, the shown payment status should be refunded
* Cancel a payment, the shown payment status should be cancelled

### Jira Links
https://jira.parkside.at/browse/WDPD-143

### CODE REVIEW

File: Tests/Integration/Model/OrderTest.php [hunk 1]
```diff
@@ -158,6 +159,27 @@ public function testIsPaymentSuccessProvider()
         ];
     }
 
+    /**
+     * @dataProvider testIsPaymentFailedProvider
+     */
+    public function testIsPaymentFailed($orderId, $isPaymentPending)
```
Comment (ps-cgrach) [inline Tests/Integration/Model/OrderTest.php]: parameter should be $bIsFailing

File: views/blocks/email_plain_order_cust_orderemail.tpl [hunk 2]
```diff
@@ -7,8 +7,11 @@
     [{oxmultilang ident="wd_wait_for_final_status"}]
   [{elseif $order->isPaymentSuccess() }]
     [{oxmultilang ident="wd_payment_success_text"}]
-  [{else}]
+  [{elseif $order->isPaymentFailed()}]
     [{oxmultilang ident="wd_order_error_info"}]
+  [{else}]
+    [{oxmultilang ident="wd_payment_awaiting"}]
```
Comment (ps-cgrach) [inline views/blocks/email_plain_order_cust_orderemail.tpl]: no `<br/>` like above?

Comment (ps-amaureder) [reply views/blocks/email_plain_order_cust_orderemail.tpl]: I will clarify it

Comment (ps-amaureder) [reply views/blocks/email_plain_order_cust_orderemail.tpl]: looks fine

**Pull request merged**